entity-service/src/test/kotlin/com/egm/stellio/entity/authorization/Neo4jAuthorizationRepositoryTest.kt

```diff
@@ -0,0 +1,249 @@
+package com.egm.stellio.entity.authorization
+
+import com.egm.stellio.entity.config.TestContainersConfiguration
+import com.egm.stellio.entity.model.Entity
+import com.egm.stellio.entity.model.Property
+import com.egm.stellio.entity.model.Relationship
+import com.egm.stellio.entity.repository.EntityRepository
+import com.egm.stellio.entity.repository.EntitySubjectNode
+import com.egm.stellio.entity.repository.Neo4jRepository
+import com.egm.stellio.entity.repository.SubjectNodeInfo
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.annotation.Import
+import org.springframework.test.context.ActiveProfiles
+
+@SpringBootTest
+@ActiveProfiles("test")
+@Import(TestContainersConfiguration::class)
+class Neo4jAuthorizationRepositoryTest {
+
+    @Autowired
+    private lateinit var neo4jAuthorizationRepository: Neo4jAuthorizationRepository
+
+    @Autowired
+    private lateinit var neo4jRepository: Neo4jRepository
+
+    @Autowired
+    private lateinit var entityRepository: EntityRepository
+
+    companion object {
+        const val EGM_CAN_READ = "https://ontology.eglobalmark.com/authorization#rCanRead"
+        const val EGM_CAN_WRITE = "https://ontology.eglobalmark.com/authorization#rCanWrite"
+        const val EGM_IS_MEMBER_OF = "https://ontology.eglobalmark.com/authorization#isMemberOf"
+        const val EGM_ROLES = "https://ontology.eglobalmark.com/authorization#roles"
+    }
+
+    @Test
+    fun `it should filter entities authorized for user with given rights`() {
+        val userEntity = createEntity(
```

Comment:
The line is longer than 120 characters ?

Reply:
Indeed, and a lot of other lines also.